luigi/scheduler.py

```diff
@@ -746,13 +820,18 @@ def get_work(self, host=None, assistant=False, current_tasks=None, worker=None,
         if assistant:
             self.add_worker(worker_id, [('assistant', assistant)])
 
+        batched_params, unbatched_params, batched_tasks, max_batch_size = None, None, [], 1
```

Comment:
`{}` is a `dict`, not a `set`.

I think it's less clear to set it to an empty dict than to set it to None. An empty dict looks like something you're going to fill rather than replace. If you think it's more clear, I'm happy to change it though.